packages/babel-plugin-transform-modules-commonjs/test/fixtures/regression/lazy-7176/output.js

```diff
@@ -1,11 +1,12 @@
 "use strict";
 
-function mod() {
+function _mod() {
   const data = babelHelpers.interopRequireWildcard(require("mod"));
-  mod = function () {
+  _mod = function () {
     return data;
   };
   return data;
 }
-mod().named;
+var mod = _mod();
+_mod().named;
```

Comment:
This is a bug, it should be `var mod = _mod;`.

It's not a bug introduced by this PR (https://github.com/babel/babel/issues/15939), however this PR makes it more common because now it's triggered also by a namespace import together with a named import (rather than just two namespace imports).

I'll see how this interacts with the lazy transform rewrite I did in https://github.com/babel/babel/pull/15878.

Reply:
https://github.com/babel/babel/pull/15941 has been merged, can you rebase it?

Reply:
> #15941 has been merged, can you rebase it?

Of course. I can do it tonight. But shouldn't I wait for #15878?

Reply:
According to my understanding, it is not required, but @nicolo-ribaudo shall prevail.

Reply:
Yes no need to wait!